cli_tools/gce_vm_image_import/importer/inflater.go

```diff
@@ -207,7 +213,15 @@ func createInflater(args ImportArguments, computeClient daisyCompute.Client, sto
 		return nil, err
 	}
 
-	return di, nil
+	if isImage(args.Source) {
+		return di, nil
+	}
+
+	ai := createAPIInflater(args, computeClient, storageClient)
+	return &inflaterFacade{
+		mainInflater:   di,
+		shadowInflater: ai,
+	}, nil
```

Comment:
Before you re-enable your feature, I'd like to see the operational plan for monitoring how it's performing in production.

If it were me, I'd do the following:

1. Add a feature guard that defaults to off, but is enabled in test
2. Get this PR approved and push it
3. Add metrics to indicate whether the API inflation was successful, how long it took, and whether the extra disk was removed
4. Observe the metrics in test
5. Submit a PR to enable the feature

I'll let you think about your plan, though, and how you want to deploy this.

Reply:
"Add a feature guard that defaults to off, but is enabled in test" -  I would leave that to the time window between PR merge and version release. In that time window, it's automatically enabled for e2e tests and disabled for users. Plus, I ran e2e tests many times from local to verify.

"Add metrics to indicate whether the API inflation was successful, how long it took, and whether the extra disk was removed"
Not necessarily a new proto field/metrics. Querying from compute-image-test-pool-001 daily to see whether there is any "shadow-disk-xxx" left is enough for our purpose.

So, the process can be:
1. merge the PR. The e2e test started to run.
2. monitor for >=3 days by query
3. release it.

It has almost the same ability as you suggested but simpler.

Rollback the PR if anything unexpected happened.

Reply:
The gate is, as before, createAPIInflater